Bottom: 93fea4a0133ee9be5198c904501a510e1624c0d0
Top:    056b8bfd424c0057063f61bff98567c021510bde
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-10-24 15:16:45 -0700

Refresh of rs-iomap

---

diff --git a/man/rsocket.7 b/man/rsocket.7
index a1171fd..5ecebc2 100644
--- a/man/rsocket.7
+++ b/man/rsocket.7
@@ -47,7 +47,7 @@ This is also known as zero-copy support, since data is sent and
 received directly, bypassing copies into network controlled buffers.
 The following calls and options support direct data placement.
 .P
-riomap
+riomap, riounmap, riowrite
 .TP
 off_t riomap(int socket, void *buf, size_t len, int prot, int flags, off_t offset)
 .TP
@@ -91,6 +91,8 @@ RDMA_SQSIZE - Integer size of the underlying send queue.
 RDMA_RQSIZE - Integer size of the underlying receive queue.
 .TP
 RDMA_INLINE - Integer size of inline data.
+.TP
+RDMA_IOMAPSIZE - Integer number of remote IO mappings supported
 .P
 Note that rsockets fd's cannot be passed into non-rsocket calls.  For
 applications which must mix rsocket fd's with standard socket fd's or
@@ -125,6 +127,8 @@ rqsize_default - default size of receive queue
 .P
 inline_default - default size of inline data
 .P
+iomap_size - default size of remote iomapping table
+.P
 If configuration files are not available, rsockets uses internal defaults.
 .SH "SEE ALSO"
 rdma_cm(7)
